src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -579,4 +579,4 @@ object Commands {
             if (args != null) function(args.asList().toTypedArray())
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Why is that here

Reply:
I had a command once, removed it in favour of /shdebug and yeah i guess thats whats left of it

Reply:
thats amazing